client/my-sites/people/viewers-list/viewers.jsx

```diff
@@ -83,6 +84,10 @@ class Viewers extends Component {
 					: this.props.translate( "You don't have any viewers yet." ),
 		};
 
+		if ( this.props.site && ! this.props.site.jetpack ) {
+			emptyContentArgs.action = <InviteButton siteSlug={ this.props.site.slug } />;
```

Comment:
I'd advise against mutating an object directly. To prevent subtle bugs, we should create a new object and assign the existing props to it. I usually suggest using spread for that:

```suggestion
			emptyContentArgs = {
				...emptyContentArgs, 
				action: <InviteButton siteSlug={ this.props.site.slug } />
			};
```